### PR TITLE
Use dockerized elastic server

### PR DESCRIPTION
**Pull request by asvetlov**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,28 +1,17 @@
...
+  - docker
 python:
-  - "3.4"
   - "3.5"
   - "3.6"
 env:
   - ES="5.5.0" AIOHTTP="2"
   - ES="5.5.0" AIOHTTP="1"
 install:
   - curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-$ES.deb && sudo dpkg -i --force-confnew elasticsearch-$ES.deb && sudo service elasticsearch start
```
Comment (hellysmile) [inline .travis.yml]: we can remove this deb at all

File: tests/conftest.py [hunk 2]
```diff
@@ -1,31 +1,103 @@
...
+        try:
+            es.exists(index='index', doc_type='doc-type', id=1)
+            break
+        except elasticsearch.TransportError as ex:
+            time.sleep(delay)
+            delay *= 2
+        finally:
+            es.transport.close()
+    else:
+        pytest.fail("Cannot start postgres server")
```
Comment (hellysmile) [inline tests/conftest.py]: elastic server

File: .travis.yml (hunk 1 above)
Comment (asvetlov) [inline .travis.yml]: Good point

File: tests/conftest.py (hunk 2 above)
Comment (asvetlov) [inline tests/conftest.py]: Ooops :)

**Pull request merged**